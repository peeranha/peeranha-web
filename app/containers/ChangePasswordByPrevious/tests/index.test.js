import { ChangePasswordByPrevious } from '../index';

const cmp = new ChangePasswordByPrevious();
cmp.props = {
  hideChangePasswordModalDispatch: jest.fn(),
  showChangePasswordModalDispatch: jest.fn(),
  children: null,
  showModal: false,
  locale: 'en',
  content: null,
  sendEmailProcessing: false,
  sendEmailDispatch: jest.fn(),
  submitEmailDispatch: jest.fn(),
  submitEmailProcessing: false,
  changePasswordDispatch: jest.fn(),
  changePasswordProcessing: false,
};

describe('<ChangePasswordByPrevious />', () => {
  it('render', () => {
    expect(cmp.render()).toMatchSnapshot();
  });
});
