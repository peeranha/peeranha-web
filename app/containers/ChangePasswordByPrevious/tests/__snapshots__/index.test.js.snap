// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ChangePasswordByPrevious /> render 1`] = `
<React.Fragment>
  <UNDEFINED
    closeModal={[MockFunction]}
    show={false}
  />
  <button
    onClick={[MockFunction]}
  />
</React.Fragment>
`;
